governance/votes/LDR_Example - Executive Vote - July 12, 2023.md

```diff
@@ -0,0 +1,174 @@
+| Harmony | [0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2](https://etherscan.io/address/0xF4704Aa4Ad22cAA2A3Dd7A7C529B4C32f7A421F2) | 1.92 |
+| VoteWizard | [0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) | 1.6 |
+| Navigator | [0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) | 0.32 |
+
+### Spark Protocol Proxy-Spell
+
+As per this [forum post](https://forum.makerdao.com/t/freeze-the-sdai-market-on-spark/21322), the Spark Protocol Proxy-Spell at [0x843A0539Ca7466Abcb769f1c1d30C8423e13A297](https://etherscan.io/address/0x843A0539Ca7466Abcb769f1c1d30C8423e13A297) will be executed if this executive proposal passes.
+
+This is a temporary measure to freeze the sDAI market. The purpose of this update is to restrict sDAI as collateral to only short ETH and staked ETH. Freezing a market prevents new deposits, but users are always free to repay/withdraw.
+
+### Blocktower/Andromeda Conduit Update
+
+As per this [forum post](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974/11) a new Output Conduit will be updated for Blocktower/Andromeda (RWA0151-A) if this executive proposal passes. This will allow the authorized party to choose with which PSM they want to interact using `hook(address psm)` function before `push` ing assets out of the conduits.
+
+For more information, the ChainSecurity audit report for **MultiSwapOutputConduit** is availible [here]([https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf 2](https://github.com/makerdao/rwa-toolkit/blob/8d30ed2cb657641253d45b57c894613e26b4ae1b/audits/ChainSecurity_MakerDAO_RWA_Toolkit_audit.pdf)).
+
+ 
+### DssCron Chainlog Housekeeping
+
+As per this [forum post](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292), the Chainlog will be brought up to date with DssCron addresses if this executive proposal passes. 
+
+#### Address Updates
+
+- Sequencer: [0x238b4E35dAed6100C6162fAE4510261f88996EC9](https://etherscan.io/address/0x238b4E35dAed6100C6162fAE4510261f88996EC9).
+- AutoLineJob [thi=1000 bps, tlo=5000 bps]: [0x67AD4000e73579B9725eE3A149F85C4Af0A61361](https://etherscan.io/address/0x67AD4000e73579B9725eE3A149F85C4Af0A61361).
+- LerpJob [maxDuration=1 day]: [0x8F8f2FC1F0380B9Ff4fE5c3142d0811aC89E32fB](https://etherscan.io/address/0x8F8f2FC1F0380B9Ff4fE5c3142d0811aC89E32fB)
+- D3MJob [threshold=500 bps, ttl=10 minutes]: [0x1Bb799509b0B039345f910dfFb71eEfAc7022323](https://etherscan.io/address/0x1Bb799509b0B039345f910dfFb71eEfAc7022323)
+- ClipperMomJob: [0xc3A76B34CFBdA7A3a5215629a0B937CBDEC7C71a](https://etherscan.io/address/0xc3A76B34CFBdA7A3a5215629a0B937CBDEC7C71a)
+- OracleJob: [0xe717Ec34b2707fc8c226b34be5eae8482d06ED03](https://etherscan.io/address/0xe717Ec34b2707fc8c226b34be5eae8482d06ED03).
+- Add FlapperJob 
```

Comment:
Why is the address missing for FlapperJob? Also, probably better to mention this as part of the Flapper changes, given it's directly related to that.